users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -798,40 +797,65 @@ def run_scf_specaug_sort():
     }
 
     nn_args, report_args_collection = get_nn_args_baseline(
-        nn_base_args={
-            "bs2x5k_scf_specaugsortlayer2": dict(
-                returnn_args={**returnn_args, "specaug_old": {"max_feature": 15, "sort_layer2": True}},
+        nn_base_args = {
+            "baseline_with_sorting": dict(
+                returnn_args={**base_returnn_args, "specaug_new": {"enable_sorting": True, "max_feature": 15, "steps_per_epoch": 4100}},
                 feature_args=feature_args,
                 lr_args=lr_args,
-                report_args={"batch_size": "2x5k"},
+                report_args={"batch_size": "2x5k", "experiment": "baseline_with_sorting"},
```

Comment:
`"experiment"` does not like a suitable descriptive header for what you are describing here. I should somehow represent that this is about SpecAugment.